engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -66,14 +65,6 @@
 	 */
 	String CONSTRAINT_MAPPING_CONTRIBUTORS = "hibernate.validator.constraint_mapping_contributors";
 
-	/**
-	 * Property corresponding to the {@link #timeProvider(TimeProvider)} method. Accepts a String with the
-	 * fully-qualified class name of a {@link TimeProvider} implementation.
-	 *
-	 * @since 5.2
-	 */
-	String TIME_PROVIDER = "hibernate.validator.time_provider";
```

Comment:
I think we should leave it in for now (circling back to our discussion about compatability). But deprecate it and remove the documentation.